lib/ZnapZend.pm

```diff
@@ -53,6 +53,7 @@ has nodelay                 => sub { 0 };
 has skipOnPreSnapCmdFail    => sub { 0 };
 has skipOnPreSendCmdFail    => sub { 0 };
 has cleanOffline            => sub { 0 };
+has mailErrorSummaryTo      => sub { q{} };
```

Comment:
```suggestion
has 'mailErrorSummaryTo';
```

Reply:
Good point; so far I was just following the precedent in nearby lines :)